### ISSUE TITLE
SD2Vita causes double launch problem (fix documented)

### ISSUE THREAD

**Issue by redion1992** [TheOfficialFloW/Adrenaline#247]
Not an issue per se, more documentation for future reference. 

I recently moved my data from ux0: on an official Sony memory card across to an SD2Vita setup I had previously set the SD to uma0: to ensure it was working, before moving all the files. However, I found that when I did, my Adrenaline bubbles required two launches to open, as if the kernel plugin was not being loaded. 

It turned out that this was caused by the load order of plugins; I had the mapping plugin in use set to load as the last kernel plugin, meaning that Henkaku was looking on (what it saw at the time as) ux0: (the Sony memory card) for the plugin, failing to find it, and moving on. It wasn’t looking on what was ux0: after boot completed (the SD2Vita), since at the time it was operating the mappings hadn’t been changed. 

The fix to this is to make sure that your SD2Vita plugin is the FIRST in the *KERNEL section, so that it is loaded and configured before any others. That way, the future plugins will be found where they’re meant to be, and are, according to VitaShell. 

TL;DR: move gamesd.skprx ABOVE adrenaline_kernel.skprx in your config.txt file.
